VeganVirus/VeganVirus/VeganVirus.vcxproj

```diff
@@ -158,6 +159,7 @@
     <ClCompile Include="Action.cpp" />
     <ClCompile Include="AudioManager.cpp" />
     <ClCompile Include="CaptureAction.cpp" />
+    <ClCompile Include="ClipBoardAction.cpp" />
```

Comment:
why is this file in the merge request?  look if you need this or wait for alon  CR for a valid opinion.  if you no need this put this in gitignore

Reply:
of course I need this. this is the file that I added